sources/api/netdog/src/net_config/mod.rs

```diff
@@ -29,6 +29,7 @@ pub(crate) trait Interfaces {
 
     /// Converts the network config into a list of `WickedInterface` structs, suitable for writing
     /// to file
+    #[allow(clippy::wrong_self_convention)]
     fn into_wicked_interfaces(&self) -> Vec<WickedInterface>;
```

Comment:
In the original PR for this code, we agreed on _not_ consuming `self`.  I don't really want to change the interface so I'm ignoring the lint.

Reply:
I think `as_$THING` is the idiomatic naming for this sort of conversion.

Reply:
^ Ahhh, yeah I like that